### DISCUSSION TITLE
Fichier Manquants

### DISCUSSION THREAD

**Discussion by Hyprolex** [SOZ-Faut-etre-Sub/SOZ-FiveM-Server#305] [Q&A] [answered]
Hello à tous !

Je me lance dans le DEV FiveM et je constate en installant le serveur qu'il me manque des fichiers qui ne sont pas présent sur le Github. Voir photo: Quelqu'un aurait la solution ?[
![Capture d'écran 2023-12-26 225111](https://github.com/SOZ-Faut-etre-Sub/SOZ-FiveM-Server/assets/46252247/390fa0fc-2eb4-49f3-840f-df33b74f3bac)
![Capture d'écran 2023-12-26 225125](https://github.com/SOZ-Faut-etre-Sub/SOZ-FiveM-Server/assets/46252247/9638edca-599b-403f-a094-db04ddc50456)
![Capture d'écran 2023-12-26 225139](https://github.com/SOZ-Faut-etre-Sub/SOZ-FiveM-Server/assets/46252247/7fa5dba2-aac1-4104-80db-8fd9b0cd7ecc)
](url)

**Comment by lanahrone** [comment]
Les fichiers ne sont pas manquants, mais doivent être compilés.
Est-ce que tu as bien build `soz-core` et `soz-phone` ? (Avec `yarn build`)

La doc : [README.md](https://github.com/SOZ-Faut-etre-Sub/SOZ-FiveM-Server?tab=readme-ov-file#prepare-and-configure-the-server)
```sh
cd resources/[soz]/soz-core && yarn install && yarn build
cd resources/[soz]/soz-phone && yarn install && yarn build
```

**Comment by Hyprolex** [comment]
![Uploading Capture d'écran 2023-12-28 164402.png…]()
Oui pardon j'avais mit les mauvais screen: voici les bons: Est§ce du à un problème de ma part ?
![Capture d'écran 2023-12-28 154419](https://github.com/SOZ-Faut-etre-Sub/SOZ-FiveM-Server/assets/46252247/46c928a2-ec2f-4397-bf6e-1a7587bfc02b)
![Capture d'écran 2023-12-28 154431](https://github.com/SOZ-Faut-etre-Sub/SOZ-FiveM-Server/assets/46252247/19099f09-3430-4cdf-ae8f-a6f8daac7775)
![Capture d'écran 2023-12-28 154442](https://github.com/SOZ-Faut-etre-Sub/SOZ-FiveM-Server/assets/46252247/2c2dd2db-d812-4516-b618-c1be81cf253c)

**Reply by lanahrone** [reply, marked as answer]
Ce ne sont que des warnings donc ça ne devrait pas t'empêcher de lancer le serveur. Il faut creuser davantage sur les ressources concernées pour voir si ça pose réellement problème, ou bien si ce ne sont que des reliquats suite à la migration vers soz-core.
